server/rest/twofishes.py

```diff
@@ -0,0 +1,115 @@
+import json
+import StringIO
+
+import geojson
+import requests
+from shapely.wkt import loads
+
+from girder.api import access
+from girder.api.describe import Description
+from girder.api.rest import Resource
+from girder.utility.model_importer import ModelImporter
+from girder.plugins.minerva.rest.geojson_dataset import GeojsonDataset
+from girder.plugins.minerva.utility.minerva_utility import findDatasetFolder
+
+
+class TwoFishes(Resource):
+    def __init__(self):
```

Comment:
@dorukozturk doing initial review. Should we add some docs for the class as well?